src/main/resources/application.properties

```diff
@@ -1,7 +1,9 @@
 #mongodb
-spring.data.mongodb.host=127.0.0.1
-spring.data.mongodb.port=27017
+spring.data.mongodb.host=localhost
+spring.data.mongodb.port=27018
```

Comment:
I would personally check your problem with hosts/ports and leave out default port